site/content/docs/0.0/getting-started/contents.md

```diff
@@ -8,4 +8,104 @@ aliases:
 toc: true
 ---
 
-{{< callout-soon "page" >}}
+## Compiled OUDS Web
+
+Once downloaded, unzip the compressed folder and you'll see something like this:
+
+<!-- NOTE: This info is intentionally duplicated in the README. Copy any changes made here over to the README too, but be sure to keep in mind to add the `dist` folder. -->
+
+```text
+@ouds/web/
```

Comment:
When you download the compressed folder, it's rather `ouds-web` than `@ouds/web`.
`@ouds/web` is the npm scope + package name.